pkg/cli/types.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	colorable "github.com/mattn/go-colorable"
+)
+
+type IO struct {
+	In  io.Reader
+	Out io.Writer
+	Err io.Writer
+}
+
+func DefaultIO() *IO {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function DefaultIO should have comment or be unexported

Reply:
fixed in 3041c36